apps/protocol-frontend/src/contexts/UserContext.tsx

```diff
@@ -587,104 +613,134 @@ export const UserContextProvider: React.FC<UserContextProps> = ({
   return (
     <UserContext.Provider
       value={{
-        userAddress,
-        setUserAddress,
-        userData,
-        setUserData,
-        userDataByAddress,
-        setUserDataByAddress,
-        userContributions,
-        setUserContributions,
-        daoContributions,
-        setDaoContributions,
-        userAttestations,
-        setUserAttestations,
-        userActivityTypes,
-        setUserActivityTypes,
         allDaos,
-        getAllDaos,
+        authenticateAddress,
+        createAttestation,
+        createContribution,
         createUser,
         createWaitlistUser,
-        createContribution,
-        createAttestation,
-        mintContribution,
+        daoContributions,
+        getAllDaos,
+        isAuthenticated,
+        isAuthenticating,
         mintAttestation,
+        mintContribution,
+        setDaoContributions,
+        setUserActivityTypes,
+        setUserAddress,
+        setUserAttestations,
+        // setUserContributions,
+        setUserData,
+        setUserDataByAddress,
         updateContribution,
-        updateProfile,
         updateLinearEmail,
-        isAuthenticated,
-        isAuthenticating,
-        authenticateAddress,
+        updateProfile,
+        userActivityTypes,
+        userAddress,
+        userAttestations,
+        userContributions,
+        userData,
+        userDataByAddress,
```

Comment:
Are these changes made by your editor? If so do you know what settings order this?

Reply:
I made the changes intentionally. It was eye straining to locate some variable, so, I sorted lines them alphabetically.